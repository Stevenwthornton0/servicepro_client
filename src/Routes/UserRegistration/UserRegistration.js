import React, { Component } from 'react';
import Registration from '../../components/Registration/Registration';

class UserRegistration extends Component {

    handleRegistrationSuccess = user => {
        const { history } = this.props;
        history.push('/login')
    }

    render() {
        return(
            <Registration onRegistrationSuccess={this.handleRegistrationSuccess}/>
        )
    }
}

export default UserRegistration;