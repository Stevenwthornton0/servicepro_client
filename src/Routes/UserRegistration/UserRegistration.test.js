import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme'
import UserRegistration from './UserRegistration';

Enzyme.configure({ adapter: new Adapter() })

describe('UserRegistration component', () => {
    it('renders without crashing', () => {
        const wrapper = shallow(<UserRegistration />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})