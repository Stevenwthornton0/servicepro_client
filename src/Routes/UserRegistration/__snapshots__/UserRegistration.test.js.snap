// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserRegistration component renders without crashing 1`] = `
<Registration
  onRegistrationSuccess={[Function]}
/>
`;
